src/views/Search.vue

```diff
@@ -0,0 +1,251 @@
+<template>
+  <div id="selenium_id_content">
+    <div
+      id="monarch-home-container"
+      class="container-fluid monarch-container">
+      <header class="intro">
+        <div class="intro-body">
+          <div class="container">
+            <div class="row">
+              <!--<div class="col-md-12">-->
+              <!--<div class="brand-heading m-0 p-0">-->
+              <!--<img-->
+              <!--class="center-block text-center img-fluid"-->
+              <!--style="max-height:80px;"-->
+              <!--src="../assets/img/monarch-logo-white-stacked.png">-->
+              <!--</div>-->
+              <!--</div>-->
+              <div
+                class="col-md-12 py-2">
+                <p class="intro-text">
+                  Search Results
+                </p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </header>
+
+      <div class="col-xs-12 col-md-9">
+        <div class="alert alert-dismissible alert-warning">FIXME</div>
+        <b-pagination
+          :total-rows="100"
+          v-model="page"
+          :per-page="25"
+          size="md"/>
+        <div class="search-results-rows">
+          <table
+            :id="'selenium_id_' + selenium_id"
+            class="search-results-table table table-striped table-sm simpletable">
+            <thead>
+              <tr>
+                <th width="25%">Term</th>
+                <th width="15%">Category</th>
+                <th width="25%">Taxon</th>
+                <th>Matching String</th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr
+                v-for="(result) in searchResults"
+                class="search-result-item">
+                <td>
+                  <router-link :to="result.toLink">
+                    {{ result.label }}
+                  </router-link>
+                </td>
+                <td>{{ result.category }}</td>
+                <td>{{ result.taxon }}</td>
+                <td v-html="result.highlight"/>
+              </tr>
+            </tbody>
+          </table>
+        </div>
+      </div>
+
+
+      <footer class="footer">
+        <home-footer/>
```

Comment:
Use FooterAll.vue instead of HomeFooter.vue. Content pages will have a thinner footer. I guess we could have a parameterized Footer.vue, but right now we have 2 files. Not a blocker for this PR.